client/core/helpers.go

```diff
@@ -140,29 +140,31 @@ func (ord *OrderReader) SettledPercent() string {
 	return ord.percent(settledFilter)
 }
 
-// FilledFrom is the sum filled in units of the outgoing asset.
+// FilledFrom is the sum filled in units of the outgoing asset. Excludes cancel
+// matches.
 func (ord *OrderReader) FilledFrom() string {
 	if ord.Sell {
-		return formatQty(ord.sumFrom(filledFilter), ord.BaseUnitInfo)
+		return formatQty(ord.sumFrom(filledNonCancelFilter), ord.BaseUnitInfo)
 	}
-	return formatQty(ord.sumFrom(filledFilter), ord.QuoteUnitInfo)
+	return formatQty(ord.sumFrom(filledNonCancelFilter), ord.QuoteUnitInfo)
```

Comment:
Looks good.  Reapplies the fix in https://github.com/decred/dcrdex/commit/890ab64885907a2416e780e2f1e2d91bd89d894b that was accidentally reverted later.